iocage/cli/create.py

```diff
@@ -53,21 +53,28 @@ def validate_count(ctx, param, value):
 
 @click.command(name="create", help="Create a jail.")
 @click.option("--count", "-c", callback=validate_count, default="1")
-@click.option("--release", "-r", required=False)
-@click.option("--template", "-t", required=False)
+@click.option("--release", "-r", required=False,
+              help="Specify the release to use for the new jail.")
+@click.option("--template", "-t", required=False,
+              help="Flag this jail as a template, which allows for rapid "
+                   "redeployment of a customized jail.")
```

Comment:
I prefer to have my spaces on the next line, so this would be:
```
              help="Flag this jail as a template, which allows for rapid"
                   " redeployment of a customized jail.")
```